Template/Tag/SentryRavenTag.web.js

```diff
@@ -0,0 +1,14 @@
+(function () {
+    return function (parameters, TagManager) {
+        this.fire = function () {
+            var dsn = parameters.get('sentryDSN');
+            if (dsn) {
+                TagManager.dom.loadScriptUrl('https://cdn.ravenjs.com/3.26.4/raven.min.js', {
```

Comment:
@Findus23 I'll merge the PR in a bit... do you know if there is maybe a URL to load the `latest` instead of a specific version?